src/index.js

```diff
@@ -37,17 +37,20 @@ const displayList = () => {
     myList.taskCollection[i].index = i;
     const listItem = document.createElement('li');
     listItem.classList.add('listItem');
-    listItem.innerHTML = `<input type="checkbox" class="checkbox" /><label contenteditable="true" class="taskDescription" type="text">${myList.taskCollection[i].description}</label><span class="deteleTask"><img src="${DeleteIcon}" class="trashCan"/></span>`;
+    listItem.innerHTML = `<input type="checkbox" class="checkbox" /><label contenteditable="true" class="taskDescription" type="text">${myList.taskCollection[i].description}</label><span class="deteleTask"><img src="${DeleteIcon}" class="trashCan" id="${i}"/></span>`;
     if (myList.taskCollection[i].completed === true) {
       listItem.classList.add('checked');
       listItem.children[0].checked = true;
     }
     list.appendChild(listItem);
   }
   checkStatus();
-  deleteTask();
   editTask();
   saveValue();
```

Comment:
- [ ] As this save task function is using local storage, you need to mock it in order to complete good tests. kindly work on it and your project should be null.

Reply:
Thanks! I made the changes, and added a test to check if it's working.